docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,178 @@
 ---
-title: Smart Optimistic Rollups
-authors: 'Nomadic Labs, TriliTech'
+title: Smart Rollups
+authors: 'Nomadic Labs, TriliTech, Tim McMackin'
 last_update:
-  date: 30 June 2023
+  date: 4 Jan 2024
 ---
 
-Rollups play a crucial part in providing next-generation scaling on Tezos. This page gives a technical introduction to Smart Rollups, their optimistic nature, and an intro to developing your own WASM kernel.
+import LucidDiagram from '@site/src/components/LucidDiagram';
 
-## Examples
+Smart Rollups play a crucial part in providing high scalability on Tezos.
+They handle transactions and logic in a separate environment that doesn't need to follow all of the same rules as the main Tezos network.
```

Comment:
I find this sentence a bit vague. The important bit for smart rollups is that the processing of transactions is performed in a separate environment. Another bit that should be stressed here is that each rollup has its own logic for processing transactions.

Also the part about not following the rules as the main Tezos network might be misleading. Rather they have dedicated rules that are enforced by the main Tezos network.

Reply:
Because this is an introduction section, I'd like to keep it vague and answer the question "What are SCORUs and why would I want to use them?"

I could combine this sentence and the next one into: "They handle logic in a separate environment that can run transactions at a much higher rate and use larger amounts of data than the main Tezos network."